docs/explanation/build-plans.rst

```diff
@@ -0,0 +1,56 @@
+Build plans
+===========
+
+A build plan is a list of platforms to build. It can be defined in a project's
```

Comment:
```suggestion
A build plan is a collection of platforms to build. It can be defined in a project's
```

Although internally the build plan is a list, I'm not sure we want to use that terminology here since it could confuse people with `platforms` being a dictionary.

Reply:
Fixed